vespaclient-java/src/main/java/com/yahoo/vespa/security/tool/CliUtils.java

```diff
@@ -16,4 +22,25 @@ public static String optionOrThrow(CommandLine arguments, String option) {
         return value;
     }
 
+    public static InputStream inputStreamFromFileOrStream(String pathOrDash, InputStream stdIn) throws IOException {
+        if ("-".equals(pathOrDash)) {
+            return stdIn;
+        } else {
+            var inputPath = Paths.get(pathOrDash);
+            if (!inputPath.toFile().exists()) {
```

Comment:
Use `Files.exists(path)` instead of converting to `File`. The latter is basically the legacy API for file management.

Reply:
Fixed here and in some other places as well.